src/open_inwoner/pdc/models/category.py

```diff
@@ -3,7 +3,19 @@
 from django.utils.translation import ugettext_lazy as _
 
 from filer.fields.image import FilerImageField
-from treebeard.mp_tree import MP_Node
+from treebeard.exceptions import InvalidMoveToDescendant
+from treebeard.mp_tree import MP_MoveHandler, MP_Node
+
+from open_inwoner.utils.managers import PublishedQueryset
```

Comment:
Usually we put model managers and querysets in the same app as related models. You can see an example in the `open_inwoner.accounts.managers` and `open_inwoner.accounts.query`
Could you move this queryset to the `accounts.pdc` folder?